runtime/compiler/z/runtime/PicBuilder.m4

```diff
@@ -1961,12 +1959,30 @@ ZZ 31 bit zOS. See definition of J9TR_CAA_SAVE_OFFSET
 
 })dnl
 
-LOAD_ADDR_FROM_TOC(r14,TR_instanceOf)
+ifdef({J9ZOS390},{dnl
 
-    BASR    r14,r14        # call instanceOf
+RestoreSSP
```

Comment:
The 31-bit path above depends on `rSSP`, which you are now restoring after that Load.  That doesn't seem correct.

```
ZZ 31 bit zOS. See definition of J9TR_CAA_SAVE_OFFSET
    L_GPR  r12,2080(rSSP)
 ```